docs/src/user_docs/assembly/io_operations.md

```diff
@@ -54,7 +54,7 @@ Memory is guaranteed to be initialized to zeros. Thus, when reading from memory
 | mem_loadw <br> - *(1 cycle)*  <br> mem_loadw.*a* <br> - *(2 cycles)*  | [a, 0, 0, 0, 0, ... ] | [A, ... ] | $A \leftarrow mem[a]$ <br> Reads a word from memory at address $a$ and overwrites top four stack elements with it. If $a$ is provided via the stack, it is removed from the stack first. <br> Fails if $a \ge 2^{32}$ |
 | mem_store <br> - *(2 cycles)*  <br> mem_store.*a*  <br> - *(3-4 cycles)*   | [a, v, ... ] | [ ... ] | $v \rightarrow mem[a][0]$ <br> Pops the top element off the stack and stores it as the first element of the word in memory at address $a$. All other elements of the word are not affected. If $a$ is provided via the stack, it is removed from the stack first. <br> Fails if $a \ge 2^{32}$ |
 | mem_storew <br> - *(1 cycle)*  <br> mem_storew.*a* <br> - *(2-3 cycles)*  | [a, A, ... ] | [A, ... ] | $A \rightarrow mem[a]$ <br> Stores the top four elements of the stack in memory at address $a$. If $a$ is provided via the stack, it is removed from the stack first. <br> Fails if $a \ge 2^{32}$ |
-| mem_stream <br> - *(2 cycles)* | [S2, S1, S0, a, ...] | [T2, T1, T0, b, ...] | $[T_0, T_1, T_2] \leftarrow permute(S_0, mem[a], mem[a+1])$ <br> $b \leftarrow a + 2$ <br> Loads two words from memory starting at the address $a$, overwrites the top 8 elements of the stack with them, and applies Rescue Prime Optimized permutation to the top 12 elements of the stack. At the end of the operation the address is incremented by $2$. |
+| mem_stream <br> - *(2 cycles)* | [C, B, A, a, ... ] | [F, E, D, a', ... ] | $[D, E, F] \leftarrow permute(A, mem[a], mem[a+1])$ <br> $a' \leftarrow a + 2$ <br> Read two sequential words from memory starting at address $a$. Subsequently, performs a Rescue Prime Optimized permutation using the 2 words read from memory as the rate portion of the hasher state, and the deepest word of the operand stack as the capacity (word A), the digest output will be in the deepest word of the operand stack (word E). |
```

Comment:
"digest output will be in the deepest word of the operand stack (word E)" should be changed to "digest output will be the second word on the operand stack (word E)."

Reply:
oh, nice catch, thansk!